source/code/plugins/changetracking_lib.rb

```diff
@@ -286,8 +286,24 @@ def self.setHash(prev_hash, last_upload_time, file_path)
                 end
     end
 
+    def self.setInventoryTimestamp(timestamp, file_path)
+        File.open(file_path, "w+", 0644) do |f| 
+             f.puts "#{timestamp}"
+        end
+    end
+
+    def self.getInventoryTimestamp(file_path)
```

Comment:
Can you append the format type to the name of API? Please add miliseconds/minutes/hours to the parameter names so that it is clear what is their scope. Example TimeInMins instead of Time.